### PR TITLE
[NO JIRA]: Fix chip colours

### PR DESCRIPTION
**Pull request by olliecurtis**
Avoids colour clashes in dark mode and on dark backgrounds

Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [ ] `README.md`
+ [x] Tests
+ [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)
+ [x] Any changes have been tested on both Android and iOS.

### CODE REVIEW

File: lib/bpk-component-chip/src/BpkChip.js [hunk 1]
```diff
@@ -72,53 +79,74 @@ const dynamicStyles = BpkDynamicStyleSheet.create({
...
   iconDisabled: {
     color: disabledTextColor,
   },
   innerOutline: {
-    backgroundColor: 'transparent',
-    borderColor: lineDarkColor,
-    borderWidth: borderSizeSm,
+    backgroundColor: {
+      light: privateChipOnDarkNormalBackgroundDay,
+      dark: privateChipDefaultNormalBackgroundNight,
```
Comment (anambl) [inline lib/bpk-component-chip/src/BpkChip.js:117]: is this right? shouldn't it be `privateChipOnDarkNormalBackgroundNight`? 🤔

File: lib/bpk-component-chip/src/BpkChip.js [hunk 2]
```diff
@@ -72,53 +79,74 @@ const dynamicStyles = BpkDynamicStyleSheet.create({
...
+    },
     paddingVertical: spacingMd - borderSizeSm,
   },
   innerOutlineSelected: {
-    backgroundColor: colorSkyBlue,
+    backgroundColor: {
+      light: privateChipOnDarkOnBackgroundDay,
+      dark: privateChipDefaultOnBackgroundNight,
```
Comment (anambl) [inline lib/bpk-component-chip/src/BpkChip.js:124]: same here - `privateChipOnDarkOnBackgroundNight`?

File: lib/bpk-component-chip/src/BpkChip.js (hunk 1 above)
Comment (olliecurtis) [reply lib/bpk-component-chip/src/BpkChip.js:117]: In dark mode in Figma the onDark is the same as the default in dark mode

File: lib/bpk-component-chip/src/BpkChip.js (hunk 2 above)
Comment (olliecurtis) [reply lib/bpk-component-chip/src/BpkChip.js:124]: In dark mode in Figma the onDark is the same as the default in dark mode

**Review by frugoman — APPROVED** [review]

**Pull request merged**